### ISSUE TITLE
[BUG] JobRunr.destroy() fails when no BackgroundJobServer

### ISSUE THREAD

**Issue by oazais** [jobrunr/jobrunr#70]
Hello, I'm having an issue here. Can you have a look?
Thanks for developing this great tool.

**Describe the bug**
When using JobRunr with no BackgroundJobServer instence JobRunr.destroy() fails.

**Environment**
I'm using version 1.2.0
I'm running on jdk-11.0.8.10-hotspot 
I'm using a SqlStorageProvider

**To Reproduce**
Creating a JobScheduler with Dashboard but not with BackgroundJobServer.
To release Dashboard server properly, I'm calling JobRunr.destroy().
But it is not released because BackgroundJobServer is NULL.
```java
@Configuration
public class JobRunrConfig {

    @Bean
    public JobScheduler initJobScheduler(StorageProvider storageProvider) {
        return JobRunr.configure()
            .useStorageProvider(storageProvider)
            .useDashboard()
            .initialize();
    }

    @PreDestroy
    public void onShutDown() {
        JobRunr.destroy();
    }
}
```

**Expected behavior**
Whatever if BackgroundJobServer or Dashboard is instentiated, a call to JobRunr.destroy() must release the resources.

**Additional context**
May be have some NULL check here?
```java
public class JobRunr {
...
    public static JobRunrConfiguration destroy() {
        jobRunrConfiguration.backgroundJobServer.stop();
        jobRunrConfiguration.dashboardWebServer.stop();
        return jobRunrConfiguration;
    }
}
```

**Comment by rdehuyss** [comment]
Already solved in v1.2.1

**Comment by oazais** [comment]
I did not see that you released a new version few days ago.
Will try it. Thanks!
